ui/package.json

```diff
@@ -17,11 +19,11 @@
     "react-app-polyfill": "^1.0.6",
     "react-copy-to-clipboard": "^5.0.2",
     "react-dom": "^16.13.1",
-    "react-overridable": "^0.0.2",
+    "react-overridable": "^0.0.3",
     "react-redux": "^7.2.0",
     "react-router-dom": "^5.1.2",
     "react-scroll": "^1.7.16",
-    "react-searchkit": "1.0.0-alpha.10",
+    "react-searchkit": "1.0.0-alpha.11",
```

Comment:
This might have fixed also https://github.com/inveniosoftware/react-invenio-app-ils/issues/355